bedrock/externalpages/templates/externalpages/pocket/tos.html

```diff
@@ -0,0 +1,213 @@
+    </section>
+
+    <section id="disclaimer">
+      <h2>Disclaimer of warranties</h2>
+      <p>Reasonable efforts are taken to improve the accuracy and integrity of the Pocket Technologies, but we are not responsible for mistakes, out-of-date information, inaccuracies, typographical or other errors. Notwithstanding any other provision of this Agreement, we reserve the right to change, suspend, remove, or disable access to any of the Pocket Technologies, content, or other materials comprising a part of the Pocket Technologies at any time without notice. In no event will we be liable for making these changes. As a registered user in good standing, you may be provided with limited access to the videos, articles or other content saved to Pocket. Such access is provided as an accommodation only, and the items you save with Pocket may be deleted at any time and for any reason. We do not warrant, and will not have any liability or responsibility for, your use of Pocket or the products or services we provide. We may also impose limits on the use of or access to certain features or portions of the Pocket Technologies, for any reason and without notice or liability. Our website, products and services may be unavailable from time to time due to mechanical, telecommunication, software, and other failures. We cannot predict or control when such downtime may occur and cannot control the duration of such downtime.</p>
+      <p><strong class="uppercase">Reasonable efforts are taken to improve the accuracy and integrity of the pocket technologies, but we are not responsible for mistakes, out-of-date information, inaccuracies, typographical or other errors. notwithstanding any other provision of this agreement, we reserve the right to change, suspend, remove, or disable access to any of the pocket technologies, content, or other materials comprising a part of the pocket technologies at any time without notice. in no event will we be liable for making these changes. as a registered user in good standing, you may be provided with limited access to the videos, articles or other content saved to pocket. such access is provided as an accommodation only, and the items you save with pocket may be deleted at any time and for any reason. we do not warrant, and will not have any liability or responsibility for, your use of pocket or the products or services we provide. we may also impose limits on the use of or access to certain features or portions of the pocket technologies, for any reason and without notice or liability. our website, products and services may be unavailable from time to time due to mechanical, telecommunication, software, and other failures. we cannot predict or control when such downtime may occur and cannot control the duration of such downtime.</strong></p>
```

Comment:
Wrong text is displayed here, you displayed the exact text as the paragraph above